app/userAuthentication.js

```diff
@@ -0,0 +1,59 @@
+import fetch from "electron-fetch/lib"
+import appInfo from "./app-info.json"
+import keytar from "keytar"
+import {BrowserWindow, session} from "electron"
+
+let mainWindow, authWindow
+
+const clientId = appInfo["client_id"]
```

Comment:
Loads clientID and clientSecret from app info json file which is configured at build time